tests/helloworld.at

```diff
@@ -0,0 +1,15 @@
+AT_BANNER([p4c compiler driver unit tests])
+
+AT_SETUP([p4c])
+
+m4_define([P4C_CHECK], [
+AT_CHECK([p4c -t tofino min.p4], [1],
```

Comment:
You should write a document describing what the compiler driver is supposed to do.
This code is really the infrastructure for building a family of compilers, one for each target. Since we don't know the needs of each target, I am not sure whether it's easy to create a driver common to all targets.